ui/espresso/MultiProcessSample/app/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright (C) 2017 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.android.testing.espresso.multiprocesssample.test">
+
+    <!-- Multi-Process Espresso only works on API level 26 and above-->
+    <uses-sdk
+        android:minSdkVersion="26"
+        android:targetSdkVersion="26"/>
+
+
+    <instrumentation
+        android:name="android.support.test.runner.AndroidJUnitRunner"
+        android:targetPackage="com.example.android.testing.espresso.multiprocesssample"
+        android:targetProcesses="*" > <!-- The instrumentation attribute introduced in O -->
+
+        <!-- The following is used by AndroidJUnitRunner to init Espresso in the new PID -->
```

Comment:
Better explanation on why this is needed and that it is mandatory for MPE to work.